src/components/table/table.tsx

```diff
@@ -41,11 +41,7 @@ const FIRST_PAGE = 1;
  * @exampleComponent limel-example-table-layout-low-density
  * @exampleComponent limel-example-table-interactive-rows
  */
-@Component({
-    tag: 'limel-table',
-    styleUrl: 'table.scss',
-    shadow: true,
-})
+@Component({ tag: 'limel-table', styleUrl: 'table.scss', shadow: true })
```

Comment:
🤔 this looks like an auto lint on save; and shouldn't be committed with your changes in this PR. I guess you should
1. make sure your branch is rebased on the latest `main`
2. run `npm i` 
3. save this file again, and see if these changes are removed from your PR